plugins/CorePluginsAdmin/templates/macros.twig

```diff
@@ -74,6 +74,11 @@
 
     <script>
         $(function () {
+            $('span.checkbox-container').on('click', function() {
+                var isAtLeastOneChecked = $('span.checkbox-container > input[checked]').length >= 1;
+                $('#update-selected-plugins').prop('disabled', !isAtLeastOneChecked);
```

Comment:
I reckon you might also need to remove the CSS class `disabled`? It's still not working for me here.

Reply:
Hello, sorry i did some mistakes with selector.